src/utils/csv.ts

```diff
@@ -0,0 +1,121 @@
+import * as Sampler from 'openapi-sampler';
+import { OpenAPIExample, OpenAPISchema } from '../types';
+import { Example } from '../types/example';
+import { MergedOpenAPISchema, OpenAPIParser } from '../services';
+
+const MAX_ITEM_DEPTH = 1;
+
+interface CsvExampleProps {
+  parser: OpenAPIParser;
+  schema: OpenAPISchema;
+  sample: OpenAPIExample;
+  samplerOptions: object;
+}
+
+const hasSameHeaders = (headers, sample) => Object.keys(sample).every(key => headers.includes(key));
```

Comment:
```suggestion
const hasSameHeaders = (headers, sample) => Object.keys(sample).every(headers.includes);
```

Reply:
I get an error:
```
TypeError: String.prototype.includes called on null or undefined
```

Reply:
```suggestion
const hasSameHeaders = (headers, sample) => Object.keys(sample).every(headers.includes.bind(headers));
```
Yeah, we have to pass headers, but now this doesn't look good. You can drop my suggestion.